PowerShell.Common.props

```diff
@@ -137,6 +137,7 @@
     <Product>PowerShell</Product>
     <Company>Microsoft Corporation</Company>
     <Copyright>(c) Microsoft Corporation.</Copyright>
+    <AssemblyTitle>PowerShell 7</AssemblyTitle>
```

Comment:
Is there a need to have this have 7 in there as the description for 5.1 is Windows PowerShell, unless you see a need for in  future to distinguish between a PowerShell 7 & a PowerShell 8 instance

Reply:
The branding of PS7 is PowerShell 7.  The current plan is to not have a situation where we'd have a PowerShell 8.  If there becomes a need for a PowerShell 8, it would mean significant breaking changes and we'd brand it PowerShell 8 vs PowerShell that is version 8.x.